src/jmespath/openFormulaFunctions.js

```diff
@@ -11,42 +11,19 @@ governing permissions and limitations under the License.
 */
 import dataTypes from './dataTypes.js';
 
-// get the offset in MS, given a date and timezone
-// timezone is an IANA name. e.g. 'America/New_York'
-function offsetMS(dateObj, timeZone) {
-  const tzOffset = new Intl.DateTimeFormat('en-US', { timeZone, timeZoneName: 'longOffset' }).format(dateObj);
-  const offset = /GMT([+\-−])?(\d{1,2}):?(\d{0,2})?/.exec(tzOffset);
-  if (!offset) return 0;
-  const [sign, hours, minutes] = offset.slice(1);
-  const result = (((hours || 0) * 60) + 1 * (minutes || 0)) * 60 * 1000;
-  return sign === '-' ? result * -1 : result;
-}
-
 function round(num, digits) {
   const precision = 10 ** digits;
   return Math.round(num * precision) / precision;
 }
 
 const MS_IN_DAY = 24 * 60 * 60 * 1000;
 
-// If we create a non-UTC date, then we need to adjust from the default JavaScript timezone
-// to the default timezone
-export function adjustTimeZone(dateObj, timeZone) {
-  if (dateObj === null) return null;
-  let baseDate = Date.UTC(
-    dateObj.getFullYear(),
-    dateObj.getMonth(),
-    dateObj.getDate(),
-    dateObj.getHours(),
-    dateObj.getMinutes(),
-    dateObj.getSeconds(),
-    dateObj.getMilliseconds(),
-  );
-  baseDate += offsetMS(dateObj, timeZone);
-
-  // get the offset for the default JS environment
-  // return days since the epoch
-  return new Date(baseDate);
+function getDateObj(dateNum) {
+  return new Date(Math.round(dateNum * MS_IN_DAY));
+}
+
+function getDateNum(dateObj) {
+  return dateObj / MS_IN_DAY;
```

Comment:
this is the same date value as returned in spreadsheet. Right ?

Reply:
> this is the same date value as returned in spreadsheet. Right ?

No...  Two reasons:
- OpenFormula does not specify an epoch.  But Excel uses: `Jan 1, 1900`, whereas we use `Jan 1, 1970`
- Excel does an 'epoch offset' where the epoch is shifted *to the local timezone*.  i.e. for a user in EST, the epoch would be `Jan 1, 1900 GMT-4:00`.  Whereas we create the value in UTC, and translate to/from the local timezone when retrieving date/time components.  

End result is that we should agree with Excel if the user calls the `hour()` function or if they do date arithmetic.  But the raw values will be different.